### PR TITLE
Color updates

### PR DESCRIPTION
**Pull request by drbyte**

### CODE REVIEW

**Comment by drbyte** [comment]
There may be more areas to refine in regard to colors. 
I'm okay with this remaining unmerged until we can review more.

**Comment by lat9** [comment]
Issue #21 opened to track these updates.

**Comment by dennisns7d** [comment]
While you're in this file, please review the header category tabs section. It appears to me that ZCA_HEADER_TABS_COLOR and ZCA_HEADER_TABS_COLOR_HOVER appear to have been swapped (perhaps since day one). Shouldn't ZCA_HEADER_TABS_COLOR apply to `#navCatTabs a` and ZCA_HEADER_TABS_COLOR_HOVER apply to `#navCatTabs a:hover`?

**Comment by drbyte** [comment]
> While you're in this file, please review the header category tabs section. It appears to me that ZCA_HEADER_TABS_COLOR and ZCA_HEADER_TABS_COLOR_HOVER appear to have been swapped (perhaps since day one). Shouldn't ZCA_HEADER_TABS_COLOR apply to `#navCatTabs a` and ZCA_HEADER_TABS_COLOR_HOVER apply to `#navCatTabs a:hover`?

Are you saying this because visually it looks wrong? Or because text-descriptions mentally seem misaligned? 

I'm having trouble seeing what you think it should look like visually.

**Comment by dennisns7d** [comment]
The text-descriptions seem misaligned.

I just looked at the admin side and found the same misalignment, so the two misalignments cancel out. Visually they are OK.

**Comment by dennisns7d** [comment]
> 
> 
> The text-descriptions seem misaligned.
> 
> I just looked at the admin side and found the same misalignment, so the two misalignments cancel out. Visually they are OK.

My old eyes don't always work. Visually they're swapped between admin and store sides.

**Comment by dennisns7d** [comment]
admin/includes/init_includes/init_bc_config.php sets the following titles for the admin side:
  ZCA_HEADER_TABS_COLOR: `Header Category Tabs Color`
  ZCA_HEADER_TABS_COLOR_HOVER: `Header Category Tabs Color on Hover`
So the admin side is aligned.

The store side is reversed. Looking at the color values on the admin side, the Color on Hover is a lower value than Color. The Color on Hover is darker than Color. Looking at the store side, the color on hover is lighter than non-hover.

**Comment by drbyte** [comment]
You're right. It's reversed, and the default colors are therefore reversed too.

**Comment by drbyte** [comment]
I've added the following to this PR:
Screen Shot 2020-12-28 at 3 49 42 PM

and PR'd an update to swap the default colors and improve the admin descriptions of those values: https://github.com/lat9/ZCA-Bootstrap-Template/pull/25/files

**Comment by drbyte** [comment]
If I've done it right, we might want to add something in the Release Notes that tells people who are upgrading to double-check (and possibly swap) the colors they've assigned for `Header Category Tabs Color` and `Header Category Tabs Color on Hover`, if they even have Category Tabs enabled.

**Review by lat9 — COMMENTED** [review]
Should we now include a `stylesheet_configs.php` (I'm sure there's a better name) to include these non-color-specific styling changes?

**Comment by drbyte** [comment]
> Should we now include a `stylesheet_configs.php` (I'm sure there's a better name) to include these non-color-specific styling changes?

IMO no.

In theory end-users won't "need" to touch/edit this file, so if it contains one entry that's not color-related but pulls dynamic values from store configuration, it won't cause any confusion.
Plus, less-is-more performance-wise when it comes to loading assets such as stylesheets. The fewer the better.

If the current naming of stylesheet_colors.php is confusing in terms of semantics, I have no objection to it being renamed.

File: includes/templates/bootstrap/css/stylesheet_zca_colors.php [hunk 1]
```diff
@@ -241,4 +246,10 @@
 .productFreePrice {
 	color: <?php echo ZCA_BODY_PRODUCTS_FREE_COLOR; ?>;
 	}
+	
+
+.legalCopyright, .legalCopyright a {color: <?php echo ZCA_FOOTER_EZPAGE_LINK_COLOR; ?>; text-decoration: none}
```
Comment (drbyte) [inline includes/templates/bootstrap/css/stylesheet_zca_colors.php:251]: I think the `text-decoration` part can be replaced with `text-decoration-none` in the template-file. I'll remove it from here.

Comment (lat9) [reply includes/templates/bootstrap/css/stylesheet_zca_colors.php:251]: OK, we'll leave the file changes and name as-is.  No sense in mangling any upgrades with under-the-cover changes.

**Pull request merged**